app/settings.py

```diff
@@ -17,22 +17,40 @@
 BASE_DIR = Path(__file__).resolve().parent.parent
 
 
+PULP_DEPLOYMENT = os.environ.get("PULP_DEPLOYMENT")
 # Quick-start development settings - unsuitable for production
 # See https://docs.djangoproject.com/en/4.0/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = os.environ.get("APP_KEY")
+SECRET_KEY = os.environ["APP_KEY"]
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
+DEBUG = PULP_DEPLOYMENT is None
 
 # let OpenShift handle that
 # (we cannot get the probes IPs from the environment and IP ranges are not supported anyway)
 ALLOWED_HOSTS = ["*"]
 
 
 # Needed for Django Admin login because OpenShift is not specifying the `Host` header
-CSRF_TRUSTED_ORIGINS = ["https://dev.analytics.pulpproject.org"]
+if PULP_DEPLOYMENT == "prod":
+    CSRF_TRUSTED_ORIGINS = ["https://analytics.pulpproject.org"]
+    PERSISTENT_MIN_AGE = 1
+    COLLECT_DEV_SYSTEMS = False
+elif PULP_DEPLOYMENT == "dev":
```

Comment:
Can we put a line of whitespace between each of these blocks for readability please?